docs/sources/setup-grafana/configure-security/configure-database-encryption/store-secrets-using-aws-secrets-manager-plugin.md

```diff
@@ -0,0 +1,123 @@
+---
+aliases:
+  - /docs/grafana/latest/setup-grafana/configure-security/configure-database-encryption/store-secrets-using-aws-secrets=manager-plugin/
+description: 'Learn how to use AWS Secrets Manager plugin to store Grafana secrets.'
+keywords:
+  - grafana
+  - secrets
+  - secretsmanager
+  - aws
+  - backend
+title: Store secrets using AWS Secrets Manager plugin
+menuTitle: Store Secrets in AWS Secrets Manager
+weight: 2000
+---
+
+# Store secrets using AWS Secrets Manager plugin
+
+By default, Grafana stores all encrypted secrets in its own SQL database. To store secrets in [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/), use the AWS Secrets Manager plugin. This is limited to datasource secrets as of Grafana version 9.1, but future versions will include all secrets, such as user credentials.
+
+You can configure options in the plugin to determine secret storage behavior in AWS Secrets Manager, and also migrate secrets from existing Grafana instances to AWS Secrets Manager. We also provide backward compatibility so that the plugin can be safely uninstalled, with some [limitations](#limitations).
+
+> **Note:** Available only in [Grafana Enterprise]({{< relref "../../../enterprise/" >}}).
+
+## Install the plugin
+
+You can install the plugin using [Grafana CLI]({{< relref "../../../cli/" >}}):
+
+```sh
+grafana-cli plugins install grafana-aws-secretsmanager
+```
+
+After you install the plugin, you must enable it.
+
+1. Open the Grafana **custom.ini** file.
+2. Locate or create the `[secrets]` section.
```

Comment:
General comment on numbering. You can number each step as `1.` Markdown numbers correctly when the docs are built.